pkg/util/collections/includes_excludes.go

```diff
@@ -120,12 +288,36 @@ func asString(in []string, empty string) string {
 	return strings.Join(in, ", ")
 }
 
+// ClusterIncludesString returns a string containing all of the cluster-scoped includes, separated by commas,
+// or <none> if the list is empty
+func (ie *IncludesExcludes) ClusterIncludesString() string {
```

Comment:
If this is only for logging, IMO we may just remove it, along with `IncludeString`, or it's totally fine to print "[]" is it's empty.

To be honest, I think it's confusing that includestring is "*" when the data is really empty.

Reply:
Deleted.

Changed to 
```
logger.Infof("Including cluster scope resources: %s", ret.clusterResourceFilter.GetIncludes())
```